### DISCUSSION TITLE
Blender Cursor Problem

### DISCUSSION THREAD

**Discussion by matthewperiut** [CuarzoSoftware/Louvre#83] [Support]
When rotating, scaling, etc after finishing the operation my mouse goes back to where it was before the operation (in louvre-views) is there a way for me to not have the mouse go back? additionally shouldn't it be precedence for it to not do that considering kde and gnome don't

**Comment by matthewperiut** [comment]
Adding additional context, just hold middle mouse and move to rotate (in the viewport), after releasing the mouse returns to where it was before pressing middle mouse.

**Comment by ehopperdietzel** [comment]
I'll review this tonight and let you know. In the meantime, I suspect Blender might be using the pointer constraints protocol to lock the cursor position. Maybe there's an option in Blender to disable it?

**Comment by ehopperdietzel** [comment]
I tested it and found that Blender is in fact using pointer constraints, so `louvre-views` is behaving as expected. It’s possible that Gnome and KDE don't support that protocol yet or are simply ignoring the constraints. 

To disable it, uncheck the `Edit > Preferences > Input > Mouse > Continuous Grab` option.
